bot/resources/tags/mcintent.md

```diff
@@ -0,0 +1,15 @@
+---
+embed:
+    title: "Message Content Intent"
+---
+
+The discord gateway only dispatches events you subscribe to, which you can configure by using "intents"
+
+The message content intent is what determines if an app will receive the actual content of newly created messages
```

Comment:
I'd just like for this to tie back to the main issue of command handlers not running
```suggestion
The message content intent is what determines if an app will receive the actual content of newly created messages. Without this intent, discord.py wouldn't know which command handler to trigger so none of your commands will work.
```